index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- Link CSS -->
+	<link rel="stylesheet" href="./styles/style.css">
+	<!-- Font Link -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+	<title>Document</title>
+</head>
+
+<body>
+
+	<header>
+		<div class="nav-bar">
+			<a href="#">Hernán Zamora</a>
+			<div class="menu">
+				<div class="bar"></div>
+				<div class="bar"></div>
+				<div class="bar"></div>
+			</div>
+		</div>
+	</header>
+
+	<main>
+		<div class="main-container">
+			<h1 class="main-title">Hey there. I'm Hernán</h1>
+			<h2 class="main-subtitle">I'am a Software Developer</h2>
+			<p class="main-text">I can help you build a product , feature or website Look through some of my work and
+				experience!
+				If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+		</div>
+		<div class="main-media">
```

Comment:
- [ ] Kindly ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; and ensure that each icon has a link (`a` tag). which will increase the accessibility of your website.

e.g

```
<ul>
  <li> 
       <a> ........ </a> 
  </li>
</ul>
```